hat/assets/js/apps/Iaso/libs/Api.ts

```diff
@@ -88,7 +90,10 @@ export const getRequest = async (
     url: string,
     signal?: Nullable<AbortSignal>,
 ): Promise<any> => {
-    return iasoFetch(url, { signal }).then(response => {
+    return iasoFetch(url, {
+        headers: { 'Accept-Language': moment.locale() },
```

Comment:
This makes me think that we should change our `LocaleProvider` to use `moment.locale()` instead of duplicating its value with `useState` can keeping both in sync. This way `moment` can effectively be the single source of truth for the locale.

@beygorghor We should discuss that